sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_generated/v3_0/_metadata.json

```diff
@@ -0,0 +1,117 @@
+{
```

Comment:
do we check this file in for multiapi?

Reply:
The `_metadata.json` files are only used when creating the generated multiapi client, since they include the info that the multiapi client would need to know about each individual API. They have no use outside of that. We shouldn't remove them from the repo though, there's precedence (see mgmt plane network) for just keeping the files